scene/animation/tween.cpp

```diff
@@ -863,6 +869,9 @@ bool Tween::seek(real_t p_time) {
 				break;
 			case INTER_CALLBACK:
 				continue;
+			default:
+				// FIXME: This switch looks weird, should be refactored.
+				break;
```

Comment:
All above changes:
```
scene/animation/tween.cpp:254:9: warning: enumeration value 'INTER_CALLBACK' not handled in switch [-Wswitch]
scene/animation/tween.cpp:287:9: warning: enumeration value 'INTER_CALLBACK' not handled in switch [-Wswitch]
scene/animation/tween.cpp:860:10: warning: enumeration value 'FOLLOW_METHOD' not handled in switch [-Wswitch]
scene/animation/tween.cpp:860:10: warning: enumeration value 'FOLLOW_PROPERTY' not handled in switch [-Wswitch]
scene/animation/tween.cpp:860:10: warning: enumeration value 'TARGETING_METHOD' not handled in switch [-Wswitch]
scene/animation/tween.cpp:860:10: warning: enumeration value 'TARGETING_PROPERTY' not handled in switch [-Wswitch]
```
@bojidar-bg @volzhs

Reply:
Line 254, 287: Good, should never be called on INTER_CALLBACK anyway. Maybe `return Variant()` + remove the `return` below the switch?
Line 860: Should be something like `if (data.type == INTER_CALLBACK) { continue; }`. Seems like this part of the code comes from the time before FOLLOW_* and TARGETING_* enums were invented.

Reply:
Thanks, will do the changes.